entrypoints/malware-scan

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+# TODO: This doesn't need the pdk, so we could consider splitting it out into another Dockerfile.
+# It does leverage the pre and post scripts though, so we'd want to keep those available.
+# Establish source dir for relative includes
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+# Pre-suite (download and extract requested release)
+# shellcheck source=../shared/entrypoint-pre disable=SC1091
+. "$DIR/../shared/entrypoint-pre"
+
+# Make sure required Anubis env vars are set
+# shellcheck source=../shared/entrypoint-anubis-pre disable=SC1091
+. "$DIR/../shared/entrypoint-anubis-pre"
+
+# FIXME: Add API key as env var
+vt_key=''
+
+# TODO: Check the tarball file size, and if larger than 32 MB, need to request an upload URL
+# at https://developers.virustotal.com/v3.0/reference#files-upload-url and use that in place
+# of the following.
+upload_url='https://www.virustotal.com/api/v3/files'
+
+upload=$(curl --request POST \
+  --url $upload_url \
+  --header "x-apikey: $vt_key" \
+  --form file=@"../$release_slug.tar.gz")
+
+result_id=$(echo $upload | jq --raw-output '.data.id')
+
+# Wait a little bit to allow the analysis to run
+sleep 60
+
+# TODO: need to loop until .data.attributes.status is "completed", or at least not "queued"
```

Comment:
The `60` here was just a value I found that worked to get completed rather than queued results. For the non-spike ticket, we'll probably want to add a loop to curl `n` times or until receiving a completed result.